javascript/reactjs-todo-journey/.env.example

```
@@ -1,14 +1,14 @@
 # VITE_APP_URL - not using this for preview-environment instead, we can use window.location.origin
 VITE_WELLKNOWN_URL=
-VITE_SCOPE=
-VITE_API_URL=
+VITE_WEB_OAUTH_CLIENT=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The VITE_WEB_OAUTH_CLIENT key should go before the VITE_WELLKNOWN_URL key

(UnorderedKey)